tests/jeos/firstrun.pm

```diff
@@ -298,7 +298,8 @@ sub run {
         create_user_in_ui();
     }
 
-    if (is_sle_micro('>6.0')) {
+    #  Base flavors come without pre-installed cockpit
```

Comment:
Nitpicks: Double space after `#` and the comment doesn't 100% match the code which checks for default instead for !base